src/experiment/trials/countdown.js

```diff
@@ -0,0 +1,64 @@
+import htmlKeyboardResponse from "@jspsych/plugin-html-keyboard-response";
+import { h1 } from "../../lib/markup/tags";
+
+/**
+ * Returns millisecond to minute
+ *
+ * @param {*} ms - millisecond
```

Comment:
Can you update the types here? It looks like they're almost always a `number`

```suggestion
 * @param {number} ms - millisecond
```